_posts/2019-07-25-Removing Easy Tables and Easy APIs from Azure App Services.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "Removing Easy Tables and Easy APIs from Azure App Services"
+author: "Ela Malani"
+tags: app service, azure app service, mobile apps
+---
+
+Mobile App Services provides specific features for Node developers to easily get started with mobile backend services by leveraging Easy Tables and Easy APIs in the Azure portal. Easy Tables provides a portal experience for Node developers to create and manage their tables, their schema, and appropriate permissions. Easy APIs lets them build and consume custom APIs in the backend. 
+
+Easy Tables and Easy APIs along with the Mobile menu in the Azure portal will be removed on **November 11, 2019** as these features have a limited audience and the existing functionality can be leveraged in alternate ways. This feature doesn’t affect developers with mobile apps in .NET backend. 
```

Comment:
I think this can be removed since .net backends are not supported in this UX anyways:

" This feature doesn’t affect developers with mobile apps in .NET backend. "